README.md

```diff
@@ -301,50 +301,17 @@ directories %w[app config lib spec your-appname/app]
 
 ## Heroku
 
-In order to deploy Ember CLI Rails app to Heroku:
+To configure your Ember CLI Rails app to be ready to deploy on Heroku:
 
-First, enable Heroku Multi Buildpack by running the following command:
+1. Run `rails g ember-cli:heroku` generator
+1. Enable Heroku Multi Buildpack by running the following commands:
 
 ```sh
+heroku config:set NPM_CONFIG_PRODUCTION=false
 heroku buildpacks:set https://github.com/heroku/heroku-buildpack-multi
```

Comment:
From the [recent docs](https://devcenter.heroku.com/articles/using-multiple-buildpacks-for-an-app#adding-a-buildpack) it looks like you can just use `heroku buildpacks:add` and don't need to declare the Multi buildpack anymore.